information-schema/information-schema-processlist.md

```diff
@@ -84,6 +108,8 @@ Fields in the `PROCESSLIST` table are described as follows:
 * `RESOURCE_GROUP`: The resource group name.
 * `SESSION_ALIAS`: The alias of the current session.
 * `ROWS_AFFECTED`: The number of rows currently affected by the statement.
+* `TIDB_CPU`: The amount of time in nanoseconds that the statement consumes the TiDB server CPU. This column shows meaningful value only when the [Top SQL](/dashboard/top-sql.md) feature is enabled. Otherwise the value will be `0`.
```

Comment:
It seems that we also need to use <CustomContent platform="tidb-cloud"> for this. Because an error occurs in the ci check for cloud links.

Reply:
Fixed